code/modules/food_and_drinks/food/snacks_other.dm

```diff
@@ -531,6 +531,33 @@
 	tastes = list("paint thinner" = 1)
 	color = "#EE35FF"
 
+/obj/item/reagent_containers/food/snacks/chewable/bubblegum/bubblegum
```

Comment:
Can you name it something more intuitive than `bubblegum/bubblegum`?

Reply:
its funny that way